### ISSUE TITLE
Create CSS for Messages Component.

### ISSUE THREAD

**Issue by toth2000** [toth2000/chatBit#8]
#### src/components/Messages/Messages.css

**Comment by calvinmorett** [comment]
@toth2000 care to help me understand what Messages.css should be targeting? Or is it just the room / chat system itself?

![image](https://user-images.githubusercontent.com/11654917/126439268-f6f7d674-cd32-4544-b62e-237e954ec0b0.png)

**Comment by toth2000** [comment]
@calvinmorett Thanks for your interest in working on the issue. 
**Message** component holds the message text and sender name that is being sent. And **Messages** Component holds all the **Message** Components Inside a scroll view. The CSS should target the respective components.

If you are interested in working on the issue, then let me know. I will assign this issue to you.

**Comment by calvinmorett** [comment]
I'm interested, but I want to make sure I understand it before diving in.

`.sentText{ background:red;} /* MY username-holder */
.pl-10{ background:#f70; } /* other-user-holder */
.messages{ background:#ffc400; } /* what is this class for? _blank space after other users message */
.messageText{background:#f2ff00; } /* username-holder */
.colorDark{background:#4f0; } /* entry message text */
.messageBox{background:#00ffc8; } /* what is this class for? */
.backgroundLight{background:#26abee; } /* what is this class for? */
.colorWhite{background:#005eff; } /* My messages */
.backgroundBlue{background:#6a00ff; } /* what is this class for? */
.messageContainer{background:#ce13af; } /* what is this class for? */
.justifyEnd{background:#ff91c1; } /* what is this class for? _blank space after my message */
`

![image](https://user-images.githubusercontent.com/11654917/126443265-bd2cfbfc-13ec-4152-a3ac-5c29d48d740f.png)

**Comment by toth2000** [comment]
@calvinmorett Please look into the following files for a better understanding:

**The whole file has a single return statement. Look inside the return statement.**

* src/components/Messages/Messages.jsx
* src/components/Message/Message.jsx

If you still have doubts then feel free to ask.

### The answer to your questions.
#### Messages Component : 
* .messages class is the parent class of all Message components. i.e it holds all the Message component (Message Text And Sender Name)

### Message Component:
* messageContainer is the parent container of all the message elements inside.
* pr-10 is padding-right = 10 and pl-10 is padding-left = 10
* messageBox is the container that holds message text
* messageText contains the message text
* the color class is for managing color

### You can rename className as per your choice if required.

**Comment by calvinmorett** [comment]
@toth2000  Please assign me this, and I'll get started on a pull request, so we can test.

**Comment by calvinmorett** [comment]
Is there a reason why on my messages, my user name is first then my message, and for other users their message is first then their username is displayed?

![image](https://user-images.githubusercontent.com/11654917/126504419-f5c4ea75-bfbd-4873-a571-440389c09173.png)

**Comment by calvinmorett** [comment]
@toth2000 How can we set it to make every user/msg pair maintain the order: username1 / message2?

**Comment by toth2000** [comment]
@calvinmorett could please explain more? I am not understanding what you are asking.

**Comment by calvinmorett** [comment]
@toth2000 Why is it alternating in the HTML?

![image](https://user-images.githubusercontent.com/11654917/126593891-fd439b7a-0089-4290-837a-74369b2758c0.png)

**Comment by toth2000** [comment]
> @toth2000 Is there a reason why on my messages, my user name is first then my message, and for other users their message is first then their username is displayed?
> 
> ![image](https://user-images.githubusercontent.com/11654917/126504419-f5c4ea75-bfbd-4873-a571-440389c09173.png)

This is the reason



> @calvinmorett To differentiate your message from others. You can modify it accordingly if you like.

**Comment by toth2000** [comment]
> @calvinmorett could please explain more? I am not understanding what you are asking.

See the code below

```
   isSentByCurrentUser ? //Check if the message is sent by current user
   //Then this block of HTML is displayed    
           (
            <div className="messageContainer justifyEnd">
                <p className="sentText pr-10">{trimmedName}</p>
                <div className="messageBox backgroundBlue">
                    <p className="messageText colorWhite">{ReactEmoji.emojify(text)}</p>
                </div>
            </div>
        )   : //Else below block of HTML is displayed
        (
            <div className="messageContainer justifyStart">
                <div className="messageBox backgroundLight">
                    <p className="messageText colorDark">{ReactEmoji.emojify(text)}</p>
                </div>
                <p className="sentText pl-10">{user}</p>
            </div>
        )
```

Hope this is what you were asking.

**Comment by calvinmorett** [comment]
Not sure if you understand my question. I'm asking what files need to be edited so that I can make the change I'm looking for?

I need it changed before I can alter the CSS. I'd rather differentiate the different users chatting in CSS, rather than having it be dictated by the HTML. I want the User class then the Message class every time, regardless of what user is chatting.

**Comment by toth2000** [comment]
@calvinmorett  Okay, yeah that a good idea.

Also, I forgot to import the CSS file into the component.
In the file components/Message/Message.jsx

add 
```
import './Message.css'
```

**Comment by calvinmorett** [comment]
@toth2000 
Please merge #10  for the Message Component. 

I've created a new issue related to the CSS that **you can assign me**, just want your opinion / feedback on the question proposed in #11

**Comment by toth2000** [comment]
> @toth2000 Please merge #10 for the Message Component.
> 
> I've created a new issue related to the CSS imports #11

Done

**Comment by toth2000** [comment]
Fixed with #19
